pkg/lib/repo/repo.go

```diff
@@ -217,26 +217,30 @@ func GetConfig(ctx context.Context, store content.Storage, configDesc ocispec.De
 }
 
 // ResolveManifest returns the manifest for a reference (tag), if present in the target store
-func ResolveManifest(ctx context.Context, store oras.Target, reference string) (*ocispec.Manifest, error) {
+func ResolveManifest(ctx context.Context, store oras.Target, reference string) (ocispec.Descriptor, *ocispec.Manifest, error) {
 	desc, err := store.Resolve(ctx, reference)
 	if err != nil {
-		return nil, fmt.Errorf("reference %s not found in remote repository: %w", reference, err)
+		return ocispec.DescriptorEmptyJSON, nil, fmt.Errorf("reference %s not found in remote repository: %w", reference, err)
```

Comment:
Should the error message change to remove "remote". I think this method can be called for local repo too.